README.md

```diff
@@ -190,6 +190,8 @@ with pool.connect() as db_conn:
 The Cloud SQL Connector for Python can be used to connect to Cloud SQL instances using both public and private IP addresses. To specify which IP address to use to connect, set the `ip_type` keyword argument Possible values are `IPTypes.PUBLIC` and `IPTypes.PRIVATE`.
 Example:
 ```python
+from google.cloud.sql.connector import IPTypes
```

Comment:
I can't suggest it, but can you change the comment on L198 to "Use private ip"

Reply:
Done